### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by Theophile11** [ugr-sail/sinergym#327]
### Question ❓

I have an error when running the DRL usage example, both the python script and the notebook example. 
The error occurs at the line
`model = DQN('MlpPolicy', env, verbose=1)`
 I provided the file showing the error, it's basically : 
 AssertionError: The algorithm only supports (<class 'gym.spaces.discrete.Discrete'>,) as action spaces but Discrete(10) was provided. 
I wondered if someone encountered this issue and how to solve it ? 

### Additional context
[drl.md](https://github.com/ugr-sail/sinergym/files/11217040/drl.md)
[](url)
Add any other context about the question here.

I found a similar issue here : https://stackoverflow.com/questions/75108957/assertionerror-the-algorithm-only-supports-class-gym-spaces-box-box-as-acti 
But I could not find a way to make it work with the solutions they proposed. 

### Checklist

- [ ] I have read the [documentation](https://ugr-sail.github.io/sinergym/compilation/html/index.html) (**required**)
- [ ] I have checked that there is no similar [issue](https://github.com/ugr-sail/sinergym/issues) in the repo (**required**)

:pencil: Please, don't forget to include more labels besides `question` if it is necessary.

**Comment by AlejandroCN7** [comment]
Hi @Theophile11!
Can you told me more information about your workspace? 

Describe the characteristic of your environment:
 * Describe how Sinergym was installed: (pip, docker, source, ...)
 * Sinergym Version: (e.g. 2.3.0)
 * OS: (Ubuntu 22.04 LTS,Dockerfile,...)
 * Python version (3.10)
 * Gymnasium version:
 * Versions of SB3:
 
 If you are using our container you don't need to give us that information. I will take a look to see if we have a problem in the tool. Thank you so much for your report.

**Comment by AlejandroCN7** [comment]
Hi @Theophile11!

We have activated Stable Baselines 3 tests in this [hotfix](https://github.com/ugr-sail/sinergym/commit/60f746a9634c7fed6ca6ab40a76e836e73e96037) (these tests were deactivated due to the fact that Stable Baselines 3 was adapting to gymnasium API). We have passed tests correctly, try to run tests and maybe it gives you information about what is wrong.

You only have to do the next:
```bash
pytest tests/test_stable_baselines.py -vvv
```
